api/swagger-spec/swagger.yml

```diff
@@ -9,6 +9,15 @@ consumes:
 schemes:
   - http
 paths:
+  /ping:
```

Comment:
@jackfrancis this path will be served by the versions API server, not this component. Currently, the HTTP server that this component runs is not defined with Swagger